app/client/src/routes/home.tsx

```diff
@@ -348,109 +358,138 @@ export function Home() {
-                  onClick={(_ev) => inputDispatch({ type: 'reset' })}
+                  className="margin-top-2 usa-button"
+                  onClick={() => setIntroVisible(false)}
                   type="button"
                 >
-                  Clear Search
+                  Close Intro
                 </button>
               </div>
-            </div>
-            <h4>Current Query</h4>
-            <CopyBox
-              text={`${window.location.origin}${window.location.pathname}/#${buildQueryString(
-                queryParams,
-              )}`}
-            />
+            </Summary>
+          )}
+          <h3>Data Profile</h3>
+          <Select
+            aria-label="Select a data profile"
+            onChange={(ev) =>
+              inputDispatch({ type: 'dataProfile', payload: ev })
+            }
+            options={listOptions.dataProfile}
+            placeholder="Select a data profile..."
+            value={dataProfile}
+          />
+          {dataProfile && (
             <>
-              <h4>{profiles[dataProfile.value].label} API Query</h4>
+              <h3>Filters</h3>
+              <FilterFields
+                dispatch={inputDispatch}
+                fields={profiles[dataProfile.value].fields}
+                options={{ ...listOptions, ...content.data.domainValues }}
+                state={inputState}
+              />
+              <h3>Download the Data</h3>
+              <div className="display-flex flex-wrap">
+                <RadioButtons
+                  legend={
+                    <>
+                      <b className="margin-right-05">File Format</b>
+                      <InfoTooltip text="Choose a file format for the result set" />
+                    </>
+                  }
+                  onChange={(option) =>
+                    inputDispatch({ type: 'format', payload: option })
+                  }
+                  options={listOptions.format}
+                  selected={format}
+                  styles={['margin-bottom-2']}
+                />
+                <div className="display-flex flex-column flex-1 margin-y-auto">
+                  <button
+                    className="align-items-center display-flex margin-x-auto margin-bottom-1 usa-button"
+                    onClick={() => null}
+                    type="button"
+                  >
+                    <Download className="height-205 margin-right-1 usa-icon width-205" />
+                    Download
+                  </button>
+                  <button
+                    className="margin-x-auto usa-button usa-button--outline"
+                    onClick={(_ev) => inputDispatch({ type: 'reset' })}
+                    type="button"
+                  >
+                    Clear Search
+                  </button>
+                </div>
+              </div>
+              <h4>
+                <GlossaryTerm term="Acidity">Current Query</GlossaryTerm>
```

Comment:
```suggestion
                {/* TODO - Remove the glossary linkage before production deployment */}
                <GlossaryTerm term="Acidity">Current Query</GlossaryTerm>
```

Can you add a TODO comment here? I just don't want this example glossary linkage to make it to production.